data/hai/hai reveal 4 middle.txt

```diff
@@ -1728,11 +1778,117 @@ mission "Hai Reveal [A12] Into the Lion's Den"
+			`	From back on your ship it is easy enough to use the station's network to place a call. You get in contact with Danforth to brief him on what's going on, with the possibility of an Unfettered fleet coming through the wormhole in a matter of days.`
+			`	He seems unsurprised, but disappointed. "When this went on as long as it did, I pulled strings to get some Cruisers specially equipped for deep-range, long-duration missions." He sighs. "This is one of those situations where I really wish I hadn't prepared for an eventuality and been right about it.`
+			`	"I'll scramble some additional Navy forces, but I can't deal with this problem personally until we finish hunting the Alphas. The Hai - even the Unfettered - are capable of restraint; they're not amoral and power-hungry like the Alphas. We need to solve the Alpha problem decisively, before they can do any real harm. After that, I'll arrange formal discussions with the Hai."`
+				decline
+
+			label "Danforth Alpha Sestor Invasion: Before Danforth"
+			action
+				log "Minor People" "Commodore Trinkenschuh" "Second in command at the Republic Navy base at Farpoint, under Admiral Danforth."
+				log `Alerted Turner and Commodore Trinkenschuh (Danforth's second-in-command) to the Hai situation. Was urged to finish finding and eliminating the Alpha enclave as soon as possible, so the Navy can deal with this new threat.`
+				log `Danforth is unavailable to help with the Hai situation due to a possible return of the Alphas from Avalon to regions north of Farpoint.`
+			`	From back on your ship it is easy enough to use the station's network to place a call. You get in contact with Farpoint's second in command, Commodore Trinkenschuh, who tells you Danforth dealing with an urgent security matter and cannot be interrupted. You brief the officer on the possibility of an Unfettered fleet coming through the wormhole in a matter of days. They promise to send enough reinforcements to the wormhole to make the Hai think twice about attacking, and arrange high-level diplomatic talks.`
+			`	When pressed for answers about why Danforth is unavailable to deal with this threat, the officer says, "There's some odd activity up north that worries us just as much: the... individuals... we found at Avalon may have returned to human space. Considering the technology near Avalon, that presents some dangerous possibilities. We'll pass this warning onto the Hai in preliminary diplomatic discussions, and try to delay our Hai problems a little while."`
+				decline
+
+
+event "Hai election has ended"
+
+
+
+mission "Hai Reveal [A12-0] Danforth Mission Sequence Fix"
+	landing
+	invisible
+	to offer
+		or
+			has "Hai Reveal [A12] Into the Lion's Den: accepted"
+			has "Hai Reveal [A12] Into the Lion's Den: done"
+		not "Danforth in mission sequence"
+		not "event: hai-human resolution announced"
+	on offer
+		# Danforth mission sequence 1 is Hai Reveal 4-6
+		"Danforth in mission sequence" = 1
```

Comment:
That was an accident, but I've tested with the txt like this. Now I'm wondering what I'll break when I fix this bug.

I think it's better to take out these lines, since I know it works when I don't fix them.

Reply:
This one is *not* a fix for continuous. It is for players to upgrade from earlier Endless Sky releases without breaking the Wanderers Campaign.